src/redux/books/books.js

```diff
@@ -1,18 +1,69 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const FETCH_BOOKS = 'bookStore/books/FETCH_BOOKS';
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/AcpJIcAvwt8ZMO0VHf3D/books';
+export const addBook = (payload) => async (dispatch) => {
+  const { id, title, author: category } = payload;
+  const response = await fetch(baseUrl, {
+    method: 'POST',
+    body: JSON.stringify({ item_id: id, title, category }),
+    headers: {
+      'content-type': 'application/json',
+    },
+  });
+  if (response.ok) {
+    dispatch({
+      type: ADD_BOOK,
+      payload,
+    });
+  } else {
+    console.log(response.statusText);
```

Comment:
- [ ] Let's clean up your code a little bit. It is not good practice to commit and push console logs.  It could expose confidential data and kill performance. Kindly remove it.